worker/client.go

```diff
@@ -306,6 +306,34 @@ func (c *Client) DownloadObject(ctx context.Context, w io.Writer, path string, o
 	return err
 }
 
+func parseContentRange(contentRange string) (start, end int64, err error) {
```

Comment:
would move this into the `api` package, rename it to `parseDownloadRange` and return an `api.DownloadRange`